index.html

```diff
@@ -2042,46 +1899,34 @@ <h3 id="applying">
         <h3>
           <code>display</code> member
         </h3>
+        <pre class="idl">
+          enum DisplayModeType {
+            "fullscreen",
+            "standalone",
+            "minimal-ui",
+            "browser"
+          };
+        </pre>
         <p>
-          The <dfn id="member-display"><code>display</code> member</dfn> is a
-          <a>string</a>, whose value is one of <a>display modes values</a>. The
-          item represents the developer's preferred <a>display mode</a> for the
-          web application. When the member is missing or erroneous, the user
-          agent MUST use the <a>fallback display mode</a>.
+          The <dfn>display</dfn> member is a <a>DisplayModeType</a>, whose
+          value is one of <a>display modes values</a>. The item represents the
+          developer's preferred <a>display mode</a> for the web application.
+          When the member is missing or erroneous, the user agent MUST use the
```

Comment:
This sentence isn't necessary; it's covered by the algorithm below (and/or the default above if one is added).

Reply:
done